avoidance/test/test_usm.cpp

```diff
@@ -0,0 +1,251 @@
+#include <gtest/gtest.h>
+
+#include <avoidance/usm.h>
+
+#include <iostream>
+
+using namespace usm;
+
+/*
```

Comment:
I thought usually you only put the copyright stuff in the header files? That being said, I realized that we seem to be missing headers everywhere

Reply:
This was copied across from another project, hopefully it is useful in more than just the PX4/Avoidance.